Data/Sys/GameSettings/RM8E01.ini

```diff
@@ -0,0 +1,492 @@
+de000000 80008180
+300c3f70 4081ff48
+140c3f68 3b9c0002
+140c41c0 3bbd0002
+140c439c 3bbd0002
+140c39b0 3bde0002
+140c3b88 3b9c0002
+141800d0 3b5a0002
+141801c0 3b9c0002
+1417f7a0 3b9c0002
+14186ff8 3b5a0002
+141870f8 3b390002
+14187184 3b9c0002
+1416be04 3b9c0002
+14173d00 3bbd0002
+1417dbb0 3b9c0002
+14177fec 38030002
+14177e4c 38030002
+14183b08 3b9c0002
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+300a7b64 4081ff48
+140a7b5c 3b9c0002
+140a7db4 3bbd0002
+140a7f90 3bbd0002
+140a75a4 3bde0002
+140a777c 3b9c0002
+14164428 3b5a0002
+14164518 3b9c0002
+14163af8 3b9c0002
+1416b350 3b5a0002
+1416b450 3b390002
+1416b4dc 3b9c0002
+1415015c 3b9c0002
+14158058 3bbd0002
+14161f08 3b9c0002
+1415c344 38030002
+1415c1a4 38030002
+14167e60 3b9c0002
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+3009ca80 4081ff48
+1409ca78 3b9c0002
+1409ccd0 3bbd0002
+1409ceac 3bbd0002
+1409c4c0 3bde0002
+1409c698 3b9c0002
+141592cc 3b5a0002
+141593bc 3b9c0002
+1415899c 3b9c0002
+141601f4 3b5a0002
+141602f4 3b390002
+14160380 3b9c0002
+14145000 3b9c0002
+1414cefc 3bbd0002
+14156dac 3b9c0002
+141511e8 38030002
+14151048 38030002
+1415cd04 3b9c0002
+e0000000 80008000
+48000000 800030C8
+DE000000 80008180
+300966F8 901F0010
+140966F0 60000000
+1409671C 60000000
+14096730 60000000
+E0000000 80008000
+48000000 800030C8
+DE000000 80008180
+30098C3C 901F0010
+14098C34 60000000
+14098C60 60000000
+14098C74 60000000
+E0000000 80008000
+48000000 800030C8
+DE000000 80008180
+300BB770 901F0010
+140BB768 60000000
+140BB794 60000000
+140BB7A8 60000000
+E0000000 80008000
+48000000 800030C8
+DE000000 80008180
+3009F30C 901F0010
+1409F304 60000000
+1409F330 60000000
+1409F344 60000000
+E0000000 80008000
+48000000 800030C8
+DE000000 80008180
+300941F8 901F0010
+140941F0 60000000
+1409421C 60000000
+14094230 60000000
+E0000000 80008000
+48000000 800030C8
+DE000000 80008180
+3009BFD4 901F0010
+1409BFCC 60000000
+1409BFF8 60000000
+1409C00C 60000000
+E0000000 80008000
+04106A24 83DE6404
+04106A28 60000000
+20744DE4 A003DE20
+04744DEC 60000000
+E2000001 80008000
+04045A78 3800FFFF
+04045468 3800FFFF
+*These Cheats implement the use of the GC Mod for Mario Party 8. This makes Mario Party 8 run via a emulated GameCube controller not a gamecube controller configured for a WiiMote.
+*http://www.mariopartylegacy.com/forum/index.php?topic=16319.0
+
+$Netplay Force Player 1 to CPU
+48000000 800030c8
+de000000 80008180
+301a0218 4bffb149
+141a0204 48000030
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+301a96d4 4bffb149
+141a96c0 48000030
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+301c6194 4bffb149
+141c6180 48000030
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+301a4954 4bffb149
+141a4940 48000030
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+301a24e8 4bffb149
+141a24d4 48000030
+e0000000 80008000
+48000000 800030c8
+de000000 80008180
+301a6ba0 4bffb149
+141a6b8c 48000030
+e0000000 80008000
+
+$Netplay Safe Kill Music
```

Comment:
This title doesn't make any sense phrased that way.

Maybe (if I'm guessing the meaning correctly)

> Kill Music in order to avoid desyncs

or something like that.

Reply:
IIRC, Netplay Safe Kill Music means that there is no music and that you won't desync if only some of the players have that code enabled. (At least for some definitions of desync... Obviously the state of the emulated console will be different in some way, but at least the intention is that there is no desync you can notice by looking at the video output.)